CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
+## v3.0.1
+
+### Map
+
+* Fixed double finger tap gesture zoom out position.
+* Added the support for automatically changing the map style to dark when entering tunnels.
+* Fixed `NavigationMapView` DocC.
+* Fixed a rare incorrect route line displaying after rerouting.
+
+### Banners and guidance instructions
+
+* Added support for the road shied name in the `WayNameView`.
+* Fixed double road name setting.
```

Comment:
Let's replace it with
```suggestion
* Fixed the current road name display.
```
to avoid the possible confusion with the "Settings"

Reply:
Thank you. Updated as "Fixed a flickering road name issue when using custom road names provided through `NavigationViewControllerDelegate.navigationViewController(_:roadNameAt:)`.". I believe it will provide more clarity.